content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/800-implicit-to-explicit-conversion.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Converting Implicit many-to-many relation to Explicit many-to-many relation'
+metaTitle: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+metaDescription: 'Learn how to convert implicit many-to-many relation to explicit many-to-many relation'
+---
+
+## Problem
+
+Users sometimes encounter situations where they need to transition from implicit many-to-many relationships between models to explicit ones. This guide provides a step-by-step walkthrough on how to make that conversion.
```

Comment:
Link implicit and explicit already here.

Reply:
I have updated the problem statement with links to implicit and explicit.